Week-3/Memorization.js

```diff
@@ -0,0 +1,37 @@
+//Create a memoize function that remembers previous inputs and stores them in cache 
+//so that itwon’t have to compute the same inputs more than once.
+ //The function will take an unspecifiednumber of integer inputs and a reducer method.
+
+
+ 
+ // Memoizer
+ const memoizeAdd = () => {
+ 
+    // Create cache for results
+    // Empty objects
+    const results = {};
+
+    // Return function which takes
+    // any number of arguments
+    return (...args) => {
+        // Create key for cache
+        const argsKey = JSON.stringify(args);
```

Comment:
Good work, What if args are empty, this will have same key for all

Just one corner case